README.rst

```diff
@@ -248,7 +248,7 @@ and connect to the REANA cloud instance where we will run this example:
 
 .. code-block:: console
 
-   $ export REANA_SERVER_URL=http://192.168.99.100:31201
+   $ export REANA_SERVER_URL=http://reana.cern.ch
```

Comment:
Since we are showing the usage with tokens lets use `https` like in the other demos.